contracts/feature-tests/use-module/tests/gov_module_whitebox_test.rs

```diff
@@ -39,58 +39,45 @@ fn world() -> ScenarioWorld {
 fn setup() -> ScenarioWorld {
     let mut world = world();
 
-    world.set_state_step(
-        SetStateStep::new()
-            .put_account(
-                OWNER_ADDRESS_EXPR,
-                Account::new()
-                    .nonce(1)
-                    .esdt_balance(GOV_TOKEN_ID_EXPR, INITIAL_GOV_TOKEN_BALANCE),
-            )
-            .new_address(OWNER_ADDRESS_EXPR, 1, USE_MODULE_ADDRESS_EXPR)
-            .put_account(
-                FIRST_USER_ADDRESS_EXPR,
-                Account::new()
-                    .nonce(1)
-                    .esdt_balance(GOV_TOKEN_ID_EXPR, INITIAL_GOV_TOKEN_BALANCE),
-            )
-            .put_account(
-                SECOND_USER_ADDRESS_EXPR,
-                Account::new()
-                    .nonce(1)
-                    .esdt_balance(GOV_TOKEN_ID_EXPR, INITIAL_GOV_TOKEN_BALANCE),
-            )
-            .put_account(
-                THIRD_USER_ADDRESS_EXPR,
-                Account::new()
-                    .nonce(1)
-                    .esdt_balance(GOV_TOKEN_ID_EXPR, INITIAL_GOV_TOKEN_BALANCE),
-            ),
-    );
+    world
+        .account(OWNER_ADDRESS)
+        .nonce(1)
+        .esdt_balance(GOV_TOKEN_ID_EXPR, INITIAL_GOV_TOKEN_BALANCE);
+    world
+        .account(FIRST_USER_ADDRESS)
+        .nonce(1)
+        .esdt_balance(GOV_TOKEN_ID_EXPR, INITIAL_GOV_TOKEN_BALANCE);
+    world
+        .account(SECOND_USER_ADDRESS)
+        .nonce(1)
+        .esdt_balance(GOV_TOKEN_ID_EXPR, INITIAL_GOV_TOKEN_BALANCE);
+    world
+        .account(THIRD_USER_ADDRESS)
+        .nonce(1)
+        .esdt_balance(GOV_TOKEN_ID_EXPR, INITIAL_GOV_TOKEN_BALANCE);
 
     // init
-    let use_module_whitebox =
-        WhiteboxContract::new(USE_MODULE_ADDRESS_EXPR, use_module::contract_obj);
-    let use_module_code = world.code_expression(USE_MODULE_PATH_EXPR);
-
-    world.whitebox_deploy(
-        &use_module_whitebox,
-        ScDeployStep::new()
-            .from(OWNER_ADDRESS_EXPR)
-            .code(use_module_code),
-        |sc| {
+    let new_address = world
+        .tx()
+        .from(OWNER_ADDRESS)
+        .raw_deploy()
+        .code(USE_MODULE_PATH_EXPR)
+        .new_address(USE_MODULE_ADDRESS)
+        .returns(ReturnsNewBech32Address)
+        .whitebox(use_module::contract_obj, |sc| {
             sc.init_governance_module(
-                managed_token_id!(GOV_TOKEN_ID),
-                managed_biguint!(QUORUM),
-                managed_biguint!(MIN_BALANCE_PROPOSAL),
+                TokenIdentifier::from(GOV_TOKEN_ID_EXPR),
```

Comment:
Let's also add a .to_token_identifier() method in TestTokenIdentifier, similar to how we have for the addresses